sentry.client.config.js

```diff
@@ -0,0 +1,9 @@
+import * as Sentry from '@sentry/nextjs';
+
+const SENTRY_DSN = process.env.SENTRY_DSN || process.env.NEXT_PUBLIC_SENTRY_DSN;
+
+Sentry.init({
+  dsn: SENTRY_DSN || 'https://388965ac235a42b990ef21332a471fc2@o204651.ingest.sentry.io/5236507',
```

Comment:
I don't think we should be committing this URL. Do we need a fallback versus using the env var?

Reply:
Yep, I'll move it to an env variable.